webapp/pages/terms-and-conditions.vue

```diff
@@ -1,36 +1,85 @@
 <template>
   <div>
     <ds-space>
-      <ds-heading tag="h2">{{ $t('site.termsAndConditions') }}</ds-heading>
+      <ds-heading tag="h2">{{ $t(`site.termsAndConditions`) }}</ds-heading>
     </ds-space>
     <ds-container>
+      <no-ssr>
+        <div>
+          <ol>
+            <li v-for="section in sections" :key="section">
+              <strong>{{ $t(`termsAndConditions.${section}.title`) }}:</strong>
+              <p v-html="$t(`termsAndConditions.${section}.description`)" />
+            </li>
+          </ol>
+          <p>{{ $t(`termsAndConditions.have-fun`) }}</p>
+          <br />
+          <p>
+            <strong v-html="$t(`termsAndConditions.closing`)" />
+          </p>
+        </div>
+      </no-ssr>
       <div>
-        <ol>
-          <li v-for="section in sections" :key="section">
-            <strong>{{ $t(`termsAndConditions.${section}.title`) }}:</strong>
-            <p v-html="$t(`termsAndConditions.${section}.description`)" />
-          </li>
-        </ol>
-        <p>{{ $t('termsAndConditions.have-fun') }}</p>
-        <br />
-        <p>
-          <strong v-html="$t('termsAndConditions.closing')" />
-        </p>
+        <ds-modal
+          v-if="isOpen"
+          v-model="isOpen"
+          v-title="$t(`site.termsAndConditionsNewTitle`)"
```

Comment:
I would prefer to have a separate page to confirm the terms and conditions. You could *link* to the current terms and conditions in the same way as on the registration page. Most users don't care and just want to check the terms and conditions to proceed. Having this page would remove clutter and let users focus on the form.